scripts/analysis.py

```diff
@@ -999,43 +967,45 @@ def stacked_bar_chart(dictionary, timestep,
             label = str(key)
         # very first country does not have a 'bottom' argument
         if sum(dictionary[key]) == 0:
-            print(str(dictionary[key]) + 'has no values')
+            print(label + ' has no values')
         elif top_index is True:
-            plot = plt.bar(left=init_year + (timestep/12),
+            plot = plt.bar(left=init_year + (timestep / 12),
```

Comment:
I feel like I already complained about this in a PR a while back. Please create a function that converts timestep to years rather than repeating the division all over the place. 

Even better, perhaps the labels could be actual dates (you have an actual start month and start year, so why not repot the actual year in the plot?)